### ISSUE TITLE
Prevent Enemy Bases from Attacking While You Attack Them

### ISSUE THREAD

**Issue by BlofeldReturns** [Anuken/Mindustry-Suggestions#4452]
### Describe the content or mechanics you are proposing.

As far as I know, at the moment, an enemy base can still attack one of your adjacent sectors, even if it is being attacked by you. I think that when you attack an enemy base, the timer that determines when the enemy base will launch an attack on your own should pause.

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

It's annoying when you launch an attack on an enemy base, and it somehow musters the strength to both attack your sector **and** defend its sector.

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by octopuscaat** [comment]
good idea but two problems. what if you just afk all the waves the the base, or just keep launching to the base to avoid them attacking you? personally this never happened to me, but i like the idea. second, possibly change the title to "prevent enemy _sectors_ from attacking while you attack them"

**Comment by BlofeldReturns** [comment]
> good idea but two problems. what if you just afk all the waves the the base, or just keep launching to the base to avoid them attacking you? personally this never happened to me, but i like the idea. second, possibly change the title to "prevent enemy _sectors_ from attacking while you attack them"

I worried about this, but... I think that that would be pretty simply solved by this feature:
"If you spend more than 15 minutes outside an active attack [on an enemy base] sector, the enemy base's attack wave counter will increase by 20."
Another option would be to just lose the match, but I thought that would be a little too harsh.
